### PR TITLE
Updating for ineligible international users

### PR DESCRIPTION
**Pull request by EllieNodder**
### Trello card

https://trello.com/c/HjAovzqh/3512-update-tta-funnel-start-page-to-address-ineligible-users-signing-up-to-rttas

### Context

The international team have flagged an issue that RTTAs are having - that they get a lot of international people signing up for an RTTA who do not have QTS and therefore are not eligible for an RTTA. We should tweak the copy to make it clearer for international users.

### CODE REVIEW

**Comment by EllieNodder** [comment]
@gemmadallmandfe Andy has reviewed and is happy with the changes. Gavin has also reviewed (although currently doesn't have access to the adviser repository so can't physically review on here). So happy to go live when you are

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -5,27 +5,35 @@
...
+        <h2 class="govuk-heading-m">Non-UK trainees and teachers</h2>
+
+        <p>To get an adviser, you'll need to have or be studying for an overseas qualification of the same standard as a UK bachelor’s degree, class 2:2 (honours) or higher.</p>
+        
+        <p>You can check the UK equivalents of your degree qualifications by calling us on <%= link_to("0800 389 2500", "tel://08003892500") %>.</p>
+
+        <p>If you're already qualified as a teacher outside the UK, you'll need to have <a href="https://www.gov.uk/guidance/qualified-teacher-status-qts">English qualified teacher status (QTS)</a> and want to teach a secondary subject to get an adviser. You can <a href="https://apply-for-qts-in-england.education.gov.uk/eligibility/start">apply for English QTS</a> if you do not already have it.</p>
+          
+        <p>Even if you do not meet the eligibility requirements for this service, you can still teach in England. <a href="https://getintoteaching.education.gov.uk/non-uk-teachers">Find more information about teaching in England for non-UK citizens</a>.</p>
```
Comment (gemmadallmandfe) [inline app/views/pages/home.html.erb:36]:
```suggestion
        <p>Even if you do not meet the eligibility requirements for this adviser service, you can still <a href="https://getintoteaching.education.gov.uk/non-uk-teachers">teach in England as a non-UK citizen</a>.</p>
```

Is it worth incorporating the link into the first sentence to shorten the paragraph?

**Review by gemmadallmandfe — COMMENTED** [review]
I wonder if this is slightly confusing now @EllieNodder - particularly separating out the (very long!) Non-UK section under its own h2, rather than being within the h2 Who is eligible section? It is also a LOT of information to get through before the Start now button, particularly for domestic ITT candidates.

Would it be clearer to split the content into something like 'If you are interested in becoming a teacher' and 'If you are already qualified as a teacher' (but worded better!)? We could then better define what TTAs and RTTAs can help with plus the eligibility criteria for each, including the relevant Non-UK stuff (eg Non-UK trainees and Non-UK teachers).

I'm very happy for you to disagree - and I am not looking to hold this up unnecessarily as this was supposed to just a quick win!

Ultimately we may need to split the funnel into two....

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me - I think this works better now it is a bit shorter!

**Pull request merged**